frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignDrawer/EditFieldDrawer/edit-fieldtype/EditDropdown/EditDropdown.tsx

```diff
@@ -106,6 +118,12 @@ export const EditDropdown = ({ field }: EditDropdownProps): JSX.Element => {
         <FormErrorMessage>
           {errors?.fieldOptionsString?.message}
         </FormErrorMessage>
+        {isFieldUsedForConditionalRouting ? (
+          <InlineMessage useMarkdown mt="1rem" variant="warning">
```

Comment:
/nit noticed that there's `useMarkdown` even though there isn't any markdown syntax being used.

Reply:
fixed in [87922cd](https://github.com/opengovsg/FormSG/pull/7804/commits/87922cd2bb7778d8f66ff9cd3ee586196f8f0fcc)